### ISSUE TITLE
Not working in Raspberry Pi 4

### ISSUE THREAD

**Issue by parthi2929** [ispysoftware/iSpy#152]
Hi

I wanted to install this in my Raspberry Pi 4., and here is my Raspbian flavor I have. 

```
pi@raspberrypi:~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```
After installing the 32 bit version of your iSpy (Agent_ARM32_3_3_6_0.zip), when I run the dot net, I get below error. 

![ispy rp4 issue 2021-05-22 15-06-54](https://user-images.githubusercontent.com/11896263/119222016-dfe28500-bb0f-11eb-95db-63c7db02eb42.png)

Suspecting the root cause could be the failing ffplay, where `/dev/videos` where not found., so I also did a `ls /dev/video*` and found they start as 1X, so this mismatch could be an issue. 

Kindly help. 

Also another issue is, every time I have to set PATH for dotnet after reboot., but I think this I should check with dotnetcore team.

**Comment by ispysoftware** [comment]
https://www.ispyconnect.com/userguide-agent-dvr.aspx#troubleshooting

**Comment by parthi2929** [comment]
I already checked [that](https://www.ispyconnect.com/userguide-agent-dvr.aspx#troubleshooting), above issue I  mentioned is not discussed there.

**Comment by ispysoftware** [comment]
The issue you have is exactly what is described in the help text - you have a missing dependency on libcvextern.so - it says it right there in the error output.

"On Linux and Raspberry Pi your system may not have some required libraries installed and Agent may fail to start with an error like libcvextern.so could not be found. You will need to use ldd to find which libraries are missing. To do this, open a terminal window, change to the Agent directory and run ldd libcvextern.so (or whatever library is causing issues). The output should tell you which dependencies need to be installed. Please let us know what's needed and what operating system you are using so we can update our docs."

**Comment by LanceMcCarthy** [comment]
This explains how to handle libcvextern issue on Rasbian OS https://www.emgu.com/wiki/index.php/Download_And_Installation#Raspberry_Pi_OS

tl;dr - it is a custom build of OpenCV for Raspberry Pi. They have the commands already configured for you so the build is really easy.
